scripts/config/org/icepear/echarts/origin/chart/treemap/breadcrumb-item-style-option.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "BreadcrumbItemStyleOption",
+  "type": "interface",
+  "extends": ["ItemStyleOption"],
+  "fields": [
+    {
+      "name": "textStyle",
+      "types": ["LabelOption"]
+    }
+  ],
+  "comments": [
+    "https://github.com/apache/echarts/blob/566ba8e83dd283ba359fe9aceee0d4a12dbcd6c7/src/chart/treemap/TreemapSeries.ts#L51",
+    "https://github.com/apache/echarts/blob/790687df55a5dbe286e52cf182c0983938efd367/src/util/types.ts#L1090"
```

Comment:
Why link this URL here?

Reply:
the second URL refers to the type of the field "LabelOption".
what URL should we explicitly include?

Reply:
Normally, we only include the URL describing this class/interface.
In some special cases e.g. we manually add some fields existing in other places, we can link the related URLs here.
The URL for LabelOption should be included in LabelOption.java not here.